ts/Dashboards/Components/KPIComponent/KPIComponent.ts

```diff
@@ -41,6 +42,12 @@ import Templating from '../../../Core/Templating.js';
 const {
     format
 } = Templating;
+
+import DashUtils from '../../Utilities.js';
+const {
+    findFirstNaN
+} = DashUtils;
+
 import U from '../../../Core/Utilities.js';
```

Comment:
If we add the `DashUtils` should we consider changing this to `CoreUtils`?

Reply:
In the future, we will have to organize everything into single shared Utils, without product division. We have such naming in many files, but we only use it at the object extracting level.